src/python/WMCore/WMRuntime/Bootstrap.py

```diff
@@ -28,6 +28,12 @@ class BootstrapException(WMException):
     """ An awesome exception """
     pass
 
+def readFloatFromFile(filePath):
+    try:
+        with open(filePath, "r") as nf:
+            return float(nf.read())
```

Comment:
AFAIK these files are supposed to have one line only, however, to be safer and protect against bad scripts/feeders, how about we read only the first line (readline)?

Reply:
I was just wondering whether there might be blank lines in front. If every file has one line I can change to readline.